src/plays/text-avatar/TextAvatar.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import UserCard from './components/UserCard';
+import { useFetchUser } from './hooks/useFetchUser';
+import './style.scss';
+const API = 'https://randomuser.me/api/?inc=name&results=10';
```

Comment:
Can we move this to a constant file and use it here?

Reply:
yes, I should change its name to `API_URL` but I think I will place it here to keep it simple

Reply:
@hiimnhan , We should keep constants in the constant file.